Documentation/contributing.rst

```diff
@@ -181,6 +181,23 @@ If for some reason, running of the provisioning script fails, you should bring t
 
     $ vagrant halt
 
+Packer-CI-Build
+^^^^^^^^^^^^^^^^
+
+In cilium, we use a custom base box with a bunch of pre-installed libraries and
+tools that we need to our daily workflow. That base box is built with `Packer
+<https://www.packer.io/>`_ and it's hosted in the `packer-ci-build
```

Comment:
... It is hosted in the `packer-ci-build <https://github.com/cilium/packer-ci-build>`_ GitHub repository.